src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import scala.collection.JavaConversions._
+import scala.collection.immutable.HashMap
+
+import org.joda.time.DateTime
+import org.scalatest.{FreeSpec, Matchers, FlatSpec}
+
+import org.broadinstitute.dsde.rawls.graph.OrientDbTestFixture
+import org.broadinstitute.dsde.rawls.model._
+
+/**
+ * Created with IntelliJ IDEA.
+ * User: hussein
+ * Date: 08/18/2015
+ * Time: 15:05
+ */
+class GraphDAODeleteSpec extends FreeSpec with Matchers with OrientDbTestFixture {
```

Comment:
FWIW there are going to _very many_ delete tests, as there are a billion edge cases. This was just the start.